libvast/src/system/spawn.cpp

```diff
@@ -79,10 +82,28 @@ expected<actor> spawn_exporter(local_actor* self, options& opts) {
     anon_send(exp, extract_atom::value, limit);
   else
     anon_send(exp, extract_atom::value);
+  if (has_continuous_option(query_opts)) {
+    self->request(self->state.tracker, infinite, get_atom::value).then(
+      [=](registry& reg) mutable {
+        VAST_DEBUG("Looking for importers");
```

Comment:
By convention, all log messages in VAST are lower-case, start with `self`, and use an active verb form. Here, this would be `VAST_DEBUG(self, "looks for importers")`.